_posts/2021-09-07-design-of-the-koji-room-koji-muro.md

```diff
@@ -0,0 +1,459 @@
+
+Our koji room, sitting at 30°C and 70% RH, will be slightly colder near the inside wall in an area known as the **temperature boundary layer**. The temperature right up against the interior wall *must* be greater than the dew point of the air in the koji room (which is 23.9°C) or condensation will form. 
+
+This is a 1-dimensional heat transfer problem. If we pick 4-inch polyurethane with a total RSI-value of 4.24, we can demonstrate that the wall temperature will not fall below 29.3°C. (see Appendix - Step 3). This might seem like overkill - our design requirement was 23.9°C! But trust me, insulation is cheap relative to the rest of the room; and the heat retention will save costs in the long run. 
+
+If the indoor conditions are maintained at 30°C, we can further demonstrate that the heat losses through the wall will total **764W**. To keep the room warm, an equal amount of heat must be supplied. Let's illustrate the conditions of the koji room at time t=0 after inoculation, during which the koji generates no heat and all openings to the room are closed:
+
+![](/assets/images/koji_room/01kojiroominitial.png)
+
+### 3. Ventilation
+
+The koji room's ventilation must be sufficient to remove excess heat, humidity, and water generated by the koji as it grows. **Removal of water vapour** is usually the limiting design criteria, and requires a greater volume of ventilation than **heat removal** or **carbon dioxide** removal.
+
+Let's demonstrate which of these three design criteria is our limit, using our example. 
+
+First, consider the requirements for **heat removal**. We can show that our 300kg of rice-koji at *hiki-komi* has a dry weight of 195kg, and will generate 1623W at the peak of its metabolism (see Appendix - Step 4). Our conductive heat losses were calculated to be 764W, so we know that ventilation for heat removal must equal 859W by exchanging indoor air with cold, outdoor air. This requires a ventilation rate of 41m³/h.
+
+
+![](/assets/images/koji_room/02kojiroomheat.png)
+
+
+Next, let's consider the requirements for **moisture removal**. About 80% of the heat generated by koji is lost as water vapour, with the remaining 20% lost due to other effects. At the peak of its heat generation, we can show that the our 300kg of koji will emit 2.16kg of water per hour (see Appendix - Step 5). By exchanging moist indoor air with dry, outdoor air, we can show that a ventilation rate of **119m³/h** is necessary to remove the moisture emitted by koji.
+
+![](/assets/images/koji_room/03kojiroomwater.png)
+
+Finally, let's consider the requirements for **carbon dioxide removal**. Koji has a respiratory coefficient of around 1.0 throughout its entire process, which means that the koji's peak heat generation of 1623W equates to a CO₂ generation rate of 0.3m³ CO₂/h (see Appendix - Step 6) using the aerobic metabolism equation:
+
+$$
+C_6H_{12}O_6+6O_2 \rightarrow 6CO_2 +6H_2O+2872kJ/\text{mol} \cdot \text{glucose}
```

Comment:
I wasn't sure what this part meant: `2872kJ/\text{mol} \cdot \text{glucose}` - so it's 2872kJ per mole of glucose? The `cdot` is throwing me off I think

Reply:
Yeah, I'm trying to depict kJ (mol-glucose)^-1 if that makes sense. The dot as in mol of glucose. Visually it makes sense to me, don't know if a simple hyphen would be better

Reply:
Let me think about this - it may just need brackets...